src/test/shared/credentials/userCredentialsUtils.test.ts

```diff
@@ -151,6 +150,11 @@ describe('UserCredentialsUtils', () => {
             assert(profiles)
             assert(profiles.credentialsFile)
             assert(profiles.credentialsFile[profileName])
+
+            const { mode } = fs.statSync(credentialsFilename)
+            const expectedMode = 0o100600
```

Comment:
Switch to constants here too

Reply:
I'm sure there is a way but I don't see many examples, and none like our use case. The constants are described as being used for checking permissions before opening and using a file. These constants are bitwise so I'm not sure the results would be as intuitive if we did use them. For the right three octals it would be something like:
`const permissions = constants. S_IRUSR  || constnts. S_IWUSR `
Then: `const fileType = constants.S_IFREG` but I'm guessing that is an octal which would need to be left shifted.. Those constant symbols are not intuitive and the bit math would only complicate the code. Unless you have a good example, I'm inclined to leave it as is.

Reply:
On Windows, only the fourth octal digit should be verified. See above comment on permissions.